### ISSUE TITLE
Var in wootility demo not showing

### ISSUE THREAD

**Issue by GuillaumeDORE** [WootingKb/wootility-issues#310]
**Describe the bug**
testing the wootility demo i encounter a var not showing: {AKC_PROFILE_MAX_COUNT}

**To Reproduce**
1. Go to 'https://wootility.io/'
2. Click on 'Touche avancé' (french version for advanced key) 
3. See error

**Screenshots / Video**
![image](https://github.com/user-attachments/assets/f87989fc-e3a0-4f9e-ad58-3e2e99dc4968)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - browser: chrome

**Wootility and Firmware version**
Find in your Wootility Settings

**Keyboard Type**
none

**Comment by simon-wh** [comment]
This is an issue with the French translation itself. At one point the core text for this section was changed, which doesn't include a number anymore.

I've removed the translation for this from the French translation on our [Crowdin project](https://crowdin.com/project/wootility). Recently we've started to use distinct translation keys when we significantly change the text of a string to prevent this from happening
